pennylane_qiskit/qiskit_device.py

```diff
@@ -181,8 +181,11 @@ def __init__(self, wires, provider, backend, shots=1024, **kwargs):
         self._capabilities["returns_state"] = self._is_state_backend
 
         # Perform validation against backend
-        backend_qubits = self.backend.configuration().n_qubits
-        # if the backend has a set number of qubits, ensure wires doesn't exceed (some simulators have n_qubits=None)
+        backend_qubits = (
```

Comment:
This change seems to suggest that we are adding support for the `BackendV2`, which is quite the opposite of what you mention in the `changelog`. Could you please confirm this once? 🤔

Reply:
Ah you're right. Sorry let me fix that. We are adding support for BackendV2.

(The context is that most Qiskit devices that are BackendV2 are actually backwards compatible with BackendV1. So on the surface it seems like the current code supports BackendV2, but actually it simply supports BackendV2s that are backwards compatible -- which is the majority of the devices at Qiskit).

Reply:
The title also says "BackendV1 support for old device API".